conda_test.sh

```diff
@@ -1,8 +1,17 @@
-PATH=`pwd`/anaconda/bin:$PATH
+#!/bin/bash
+set -x
+set -e
+source conda_env.sh
 
-export DYLD_FALLBACK_LIBRARY_PATH=`pwd`/anaconda/lib:`pwd`/anaconda/lib/python2.7/site-packages/itaps:$DYLD_FALLBACK_LIBRARY_PATH
-export DYLD_LIBRARY_PATH=`pwd`/anaconda/lib:`pwd`/anaconda/lib/python2.7/site-packages/itaps:$DYLD_LIBRARY_PATH
+tar_dir="pyne-ci"
 
-`pwd`/anaconda/bin/nosetests -w ./pyne/tests
+# Setup workdir
+if [ -d "${BLD}/work/${tar_dir}" ]; then
+  export WORKDIR="${BLD}/work/${tar_dir}"
+else  
+  export WORKDIR="${BLD}/work"
+fi
 
-exit $?
+cd "${WORKDIR}/tests"
+nosetests -vs
```

Comment:
I have seen hard-to-debug issues with this pattern in Cyclus, may I suggest `nosetests -vsw ${WORKDIR}/tests`?